arrows/ffmpeg/tests/test_video_input_ffmpeg.cxx

```diff
@@ -323,188 +311,164 @@ TEST_F ( ffmpeg_video_input, read_video_aphill )
 // ----------------------------------------------------------------------------
 TEST_F ( ffmpeg_video_input, read_video )
 {
-  // make config block
-  kwiver::arrows::ffmpeg::ffmpeg_video_input input;
-
-  kwiver::vital::path_t correct_file = data_dir + "/" + video_name;
-
-  input.open( correct_file );
-
-  kwiver::vital::timestamp ts;
+  ffmpeg::ffmpeg_video_input input;
+  input.open( ffmpeg_video_path );
 
-  EXPECT_EQ( TOTAL_NUMBER_OF_FRAMES, input.num_frames() ) <<
-    "Number of frames before extracting frames should be " <<
-    TOTAL_NUMBER_OF_FRAMES;
+  EXPECT_EQ( expected_frame_count, input.num_frames() )
+    << "Number of frames before extracting frames should be "
+    << expected_frame_count;
 
-  int num_frames = 0;
+  kv::timestamp ts;
+  size_t frame_count = 0;
   while( input.next_frame( ts ) )
   {
+    ++frame_count;
+
     auto img = input.frame_image();
     auto md = input.frame_metadata();
 
-    ++num_frames;
-    EXPECT_EQ( num_frames, ts.get_frame() ) <<
-      "Frame numbers should be sequential";
-    EXPECT_EQ( ts.get_frame(), decode_barcode( *img ) ) <<
-      "Frame number should match barcode in frame image";
+    EXPECT_EQ( frame_count, ts.get_frame() )
+      << "Frame numbers should be sequential";
+    EXPECT_EQ( ts.get_frame(), decode_barcode( *img ) )
+      << "Frame number should match barcode in frame image";
 
     EXPECT_EQ( red, test_color_pixel( 1, *img ) );
     EXPECT_EQ( green, test_color_pixel( 2, *img ) );
     EXPECT_EQ( blue, test_color_pixel( 3, *img ) );
   }
-  EXPECT_EQ( TOTAL_NUMBER_OF_FRAMES, num_frames ) <<
-    "Number of frames found should be " <<
-    TOTAL_NUMBER_OF_FRAMES;
-  EXPECT_EQ( TOTAL_NUMBER_OF_FRAMES, input.num_frames() ) <<
-    "Number of frames after extracting frames should be " <<
-    TOTAL_NUMBER_OF_FRAMES;
+  EXPECT_EQ( expected_frame_count, frame_count )
+    << "Number of frames found should be " << expected_frame_count;
```

Comment:
Maybe also add the `frame_count` to this message?
```suggestion
    << "Number of frames found should be " << expected_frame_count
    << " but found " << frame_count'
```

Reply:
I would actually be more inclined to remove these messages altogether, since the values of both items being compared are always printed by GTest when the assertion fails.

Reply:
That sounds good to me, I would consider that also resolving the other comments